### PR TITLE
Using OpenShift 4.12 on Jenkins CI Jobs

### PR DESCRIPTION
**Pull request by otaviof**

### CODE REVIEW

File: ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ images:
   dockerfile_path: Dockerfile.rhel8
   from: base
   inputs:
-    ocp_builder_rhel-8-golang-1.18-openshift-4.11:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.12:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11
```
Comment (jkhelil) [inline ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml:27]: you need to change it here also

File: ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ images:
...
```
Comment (jkhelil) [inline ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml:39]: and here

**Review by jkhelil — CHANGES_REQUESTED** [review]
it is also used in this file in other places
 container:
    clone: true
    from: ocp_builder_rhel-8-golang-1.17-openshift-4.11
  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
- as: e2e-jenkins-image-tests
  commands: IMAGE_NAME=stable:jenkins make smoke
  container:
    clone: true
    from: ocp_builder_rhel-8-golang-1.17-openshift-4.10

**Comment by jkhelil** [comment]
No need to do these changes for 4.11

File: ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml (hunk 1 above)
Comment (otaviof) [reply ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml]: Thanks! I've changed more references, please consider the latest changes.

**Comment by jkhelil** [comment]
@otaviof 
I think you still need to the changes in these places


container:
clone: true
from: **ocp_builder_rhel-8-golang-1.17-openshift-4.11**
skip_if_only_changed: ^docs/|.md$|^(?:.*/)?(?:.gitignore|OWNERS|PROJECT|LICENSE)$

as: e2e-jenkins-image-tests
commands: IMAGE_NAME=stable:jenkins make smoke
container:
clone: true
from: **ocp_builder_rhel-8-golang-1.17-openshift-4.10**

**Pull request merged**